src/lib/input-validation/utils/regexGenerator.ts

```diff
@@ -120,8 +120,12 @@ export const generatePattern = ({
   isCustomRegex,
   specialCharacters,
 }: RegexGeneratorParam): string => {
-  if (isCustomRegex) return regex;
-  if (allowUnicode) return UNICODE_PATTERN;
+  if (isCustomRegex) {
```

Comment:
just curious, why do we add curly braces here?

Reply:
suddenly eslint show an error and want me to add curly braces, it works on my previous PR though